arex-schedule-web-api/src/main/java/com/arextest/schedule/service/PlanProduceService.java

```diff
@@ -257,5 +264,45 @@ private BuildReplayFailReasonEnum validateToResultReason(BuildPlanValidateResult
         }
     }
 
+    public CommonResponse reRunPlan(String planId) {
+        ReplayPlan replayPlan = replayPlanRepository.query(planId);
+        if (replayPlan == null) {
+            return CommonResponse.badResponse("target plan not found");
+        }
+        int rerunStatus = StageUtils.getStageStatus(replayPlan.getReplayPlanStageList(), PlanStageEnum.RE_RUN);
+        if (rerunStatus == StageStatusEnum.PENDING.getCode() || rerunStatus == StageStatusEnum.ONGOING.getCode()) {
+            return CommonResponse.badResponse("This plan is ReRunning");
+        }
+        replayPlan.setReRun(Boolean.TRUE);
```

Comment:
can we add this field to BizLog, need to seperate rerun logs from regular logs to avoid confusion